common/workunit/workunit.cpp

```diff
@@ -5417,7 +5417,9 @@ void CWorkUnitFactory::importWorkUnit(const char *zapReportFileName, const char
                 fileNameWithPath.append(PATHSEPSTR).append(pathTail(fileName));
                 if (checkFileExists(fileNameWithPath)) //Check if the ZAP report contains this file.
                 {
-                    fileTree.setProp("@ip", localIP);
+#ifndef _CONTAINERIZED
+                    fileTree.setProp("@ip", GetCachedHostName());
```

Comment:
question: perhaps this should set to "localhost" unconditionally in containerized mode?

It doesn't need setting if the zap report was exported from the cloud, where these will already be localhost, but if the zap report is from bare metal, than they will need resetting to localhost to be accessible.

Does that work, importing a zap report from bare-metal to cloud ? I can see it's usefulness

Reply:
Now, the ip is set to "localhost" unconditionally in containerized mode. With this change, I did import a zap report from bare-metal to cloud. I can see the graph, results, Query Associated Files, etc. But, I need to fix the process log (in https://track.hpccsystems.com/browse/HPCC-27855?).